src/Setup.elm

```diff
@@ -26,27 +26,11 @@ type Msg
     | DistrictFactor Float
     | PlayOffFactor Float
     | OffSeasonFactor Float
-    | Continue
 
 
-main : Program () Model Msg
-main =
-    Browser.sandbox
-        { init = init
-        , view = view
-        , update = update
-        }
-
-
-init : Model
+init : Float -> Float -> Float -> Float -> Int -> Int -> Model
 init =
-    { einsteinFactor = 1
-    , districtFactor = 1
-    , playOffFactor = 1
-    , offSeasonFactor = 1
-    , slider1 = 2017
-    , slider2 = 2019
-    }
+    Model
```

Comment:
This signature is too easy to confuse.
Are you sure you need this much customization?
If you do, receive a record to eliminate order-induced bugs.